### ISSUE TITLE
IT80-002 doesn't align or resize images and qr after .set

### ISSUE THREAD

**Issue by franvera** [python-escpos/python-escpos#109]
Printer equal IT80-002 ESC/POS

```python
#!/usr/bin/env python
# -*- coding: utf-8 -*- 
from escpos import *
from time import sleep

Epson = printer.File("/dev/usb/lp0")

Epson.set("LEFT", "A", "B", 1, 1)
Epson.text("Left\n")
Epson.qr("test")
Epson.image("test.jpg")

Epson.set("RIGHT", "A", "B", 1, 1)
Epson.text("Right\n")
Epson.qr("test")
Epson.image("test.jpg")

Epson.set("CENTER", "A", "B", 2, 2)
Epson.text("CENTER DOUBLE\n")
Epson.qr("test")
Epson.image("test.jpg")

Epson.cut()
```
![camera_capture](https://cloud.githubusercontent.com/assets/4934622/13823534/8c721388-eb6f-11e5-9c1c-279c761d9d31.jpg)

Any ideas? Thanks for the help

**Comment by patkan** [comment]
I am sorry, but I can't reproduce. I have adapted your script for my serial printer (Serial instead of File) and have removed the image printing. The alignment of the QR-code prints just fine.

To rule out that I have "removed" the problem, could you please remove the image-printing line and see if it persists?
Moreover you could try to call the method flush() after setting the alignment. (the file printer has problems with buffered commands, we will probably have to call flush automatically in the future) However it seems that the printer receives the command.

As a next step you could try to acquire the command reference for your specific model, try to construct the relevant commands "manually" and then send them with the _raw() method. This way you can be sure that the printer receives the right commands.

**Comment by DavisGoglin** [comment]
The command in the documentation linked at mike42/escpos-php/issues/96 is identical to the epson documentation. So we are sending the right command according to docs. 

Maybe see if 

    Epson._raw(b'\x1b\x61\x32')
    Epson.text("Right\n")
    Epson.image("test.jpg")

aligns it right. It's using the alternative: `ESC + a + 50` instead of `ESC + a + 2`

**Comment by franvera** [comment]
Thanks @DavisGoglin , unfortunately the problem persists. It worked for text but not for the images.

**Comment by franvera** [comment]
The problem was resolved on the referenced issue using the ESC * command for printing images.

**Comment by DavisGoglin** [comment]
For the record here, it was solved at mike42/escpos-php#96  by using [ESC *](https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=88) bit image mode column format printing. 

We are using the obsolete command `GS v 0` (bit image raster mode) to print instead of `ESC *` (bit image column mode), or the recommended image command `GS ( L` (set graphics data).
